docs/device_reference.md

```diff
@@ -2,16 +2,18 @@
     - Fire OS 5: Based on Android 5.1 (Lollipop, API level 22)
     - Fire OS 6: Based on Android 7.1 (Nougat, API level 25)
 
-There are currently two devices:
+There are currently three devices:
 
-| Series | Release | Code  | OS | Size  | Resources | DPI  | Emulator  | Resolution |
-|:------:|:-------:|:-----:|:--:|:-----:|:---------:|:----:|:---------:|:----------:|
-| 5      | 2019    | AEOCH | ?  | ?     | large     | ?    | ?         | 960x480    |
-| 2      | 2018    | AEOBP | 5  | 10.1" | xlarge    | mdpi | WXGA 10.1 | 1280x800   |
-| 1      | 2017    | AEOKN | 5  | 7"    | large     | mdpi | ?         | 1024x600   |
+| Series | Release | Code  | OS | Size  | Resources | DPI  | Emulator  | Resolution | Available Height |
+|:------:|:-------:|:-----:|:--:|:-----:|:---------:|:----:|:---------:|:----------:|:-----------------|
+| 5      | 2019    | AEOCH | ?  | ?     | large     | ?    | ?         | 960x480    | 456dp
+| 2      | 2018    | AEOBP | 5  | 10.1" | xlarge    | mdpi | WXGA 10.1 | 1280x800   | 770dp
+| 1      | 2017    | AEOKN | 5  | 7"    | large     | mdpi | ?         | 1024x600   | 570dp
 
 We recommend **developing** on Android tablet emulators for developer efficiency: the recommended Android emulator images are mentioned above.
 
+Note that available height does not match actual height.  See [documentation][res height] for details, but it seems that some space is reserved for toolbars and similar "screen decorations."  These values were retrieved by querying resources.configuration.screenHeightDp on device.
```

Comment:
The paragraph above and below this are related so move this below the paragraph below. Also, given this is related to "Available Height", maybe you should make it, "Available Height*" and "*: Available height does not match actual height."